### PR TITLE
Update branding/sort on aggregaotors; add page on selecting OS

### PR DESCRIPTION
**Pull request by Ottergoose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
 # ADS-B Reception, Decoding & Sharing with Docker
 
-Automatic Dependent Surveillance-Broadcast \(ADS-B\) is a surveillance technology in which an aircraft determines its position via satellite navigation and periodically broadcasts it, enabling it to be tracked.
+Automatic Dependent Surveillance-Broadcast \(ADS-B\) is a safety and surveillance technology in which an aircraft determines its position via satellite navigation and periodically broadcasts it, enabling it to be tracked.
 
-This ADS-B data can be received by ~~nerds~~ enthusiasts using Software Defined Radio \(SDR\), and used for fun and profit. For example:
+These ADS-B data can be received by ~~nerds~~ enthusiasts using Software Defined Radio \(SDR\), and used for fun and profit.
```
Comment (mikenye) [inline README.md:5]: I think it should be "This" as-per the original.

"These ADS-B data" sounds.... weird. :)

We need an English Major :P

**Review by mikenye — CHANGES_REQUESTED** [review]
Please take a look at the comment on README.md.

Thanks.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
-* "New" aggregator services such as [adsb.fi](https://globe.adsb.fi/), [adsb.one](https://adsb.one/), and [adsb.lol](https://adsb.lol/)
 
 **Profit:**
 
-* [https://flightaware.com/adsb/piaware/](https://flightaware.com/adsb/piaware/)
-* [https://www.flightradar24.com/share-your-data](https://www.flightradar24.com/share-your-data)
-* [https://planefinder.net](https://planefinder.net)
-* [https://www.radarbox.com](https://www.radarbox.com)
-* [https://radarvirtuel.com](https://radarvirtuel.com)
 * [ADS-B Exchange](https://adsbexchange.com/)
```
Comment (tedder) [inline README.md:20]: TBH my vote is to not alphasort adsbexchange. Just my opinion.

Comment (mikenye) [reply README.md:20]: Agree 100% 💯

File: README.md (hunk 1 above)
Comment (Ottergoose) [reply README.md:5]: It's weird but correct; the word "data" is plural like "chickens" or "geese." Whether or not to always treat it as a plural is a matter of preference.

File: README.md (hunk 2 above)
Comment (Ottergoose) [reply README.md:20]: What's the preferred method of sorting them?

Comment (mikenye) [reply README.md:20]: Alphabetical is ok, but stick ADSB Exchange at the bottom. Given recent events I don't really want to promote them...

**Review by mikenye — APPROVED** [review]

**Comment by mikenye** [comment]
Thanks @Ottergoose and other contributors!

**Pull request merged**